app/contacts/page.js

```diff
@@ -0,0 +1,48 @@
+'use client'
+import styles from '../page.module.css'
+import { ContactsList } from '../components/contactsList'
+import { contactsApi } from '../components/ContactsApi'
+import { SearchBar } from '../components/SearchBar'
+import { useState } from 'react'
+import { NewContactButton } from '../components/NewContactButton'
+
+
+export default function Contacts() {
+  const [contacts, _] = useState(contactsApi.all())
+  const [filteredContacts, setFilteredContacts] = useState(contactsApi.all())
+
+  const contactsSearch = (term) => {
+    // setFilteredCOntacts to what's returned from the filter
+
+    if (!term.length) {
+      setFilteredContacts(contacts)
+    }
+
+    console.log({ term: term })
+
+
+    const lowerCaseSearchTerm = term.toLowerCase()
```

Comment:
you should also [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim), 
Will prevent using spaces in your search box.